SECURITY.md

```diff
@@ -1,66 +1,17 @@
-# Security Policy
+# Reporting Security Vulnerabilities
```

Comment:
I'm not sure if we're supposed to replace the upstream SECURITY.md if it exists. In the OLCNE repos the latest practice is to keep the existing one and create a new one name `ORACLE-SECURITY.md` or something like that, I think? I'm not sure what the right thing to do is, but we should check and confirm if you haven't already.

Reply:
I think these changes were made during 2.6.8 release. Do we need to backport the latest practice to 2.6.8 if it's confirmed?